components/form/form-errory-summary.js

```diff
@@ -0,0 +1,109 @@
+import '../alert/alert.js';
+import '../link/link.js';
+import '../expand-collapse/expand-collapse-content.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodyStandardStyles } from '../typography/styles.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class FormErrorSummary extends LocalizeCoreElement(RtlMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			errors: { type: Object, attribute: false },
+			_expanded: { type: Boolean, attribute: false }
+		};
+	}
+
+	static get styles() {
+		return [bodyStandardStyles, css`
+
+			:host {
+				display: block;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			.d2l-form-error-summary-header {
+				align-items: top;
+				cursor: pointer;
+				display: flex;
+				justify-content: space-between;
+				padding: 0.3rem 0.3rem 0.3rem 1.2rem;
+			}
+
+			:host([dir="rtl"]) .d2l-form-error-summary-header {
+				padding-left: 0.3rem;
+				padding-right: 1.2rem;
+			}
+
+			.d2l-form-error-summary-text {
+				align-items: center;
+				display: flex;
+			}
+
+			.d2l-form-error-summary-error-list {
+				margin: 0 0 0 1.2rem;
+				padding-bottom: 0.65rem;
+				padding-top: 0.3rem;
+			}
+
+			:host([dir="rtl"]) .d2l-form-error-summary-error-list {
+				margin-left: 0;
+				margin-right: 1.2rem;
+			}
+		`];
+	}
+
+	constructor() {
+		super();
+		this.errors = [];
+		this._expanded = true;
+	}
+
+	render() {
+		const errorSummaryTerm = this.errors.length === 1 ? 'components.form-error-summary.singleError' : 'components.form-error-summary.multipleErrors';
```

Comment:
Ooooh, this looks like a fun opportunity to use [the "plural" keyword](https://formatjs.io/docs/icu-syntax#plural-format) in the Intl Message syntax!

Reply:
I did consider that approach but I'm a bit apprehensive to use it because the more complex syntax often gets mangled during translation. But I can definitely change it if we're not concerned about that.

Reply:
I was actually interested in running that experiment to see what we got back haha...